packages/lit-dev-content/site/docs/v3/tools/testing.md

```diff
@@ -75,3 +75,48 @@ export default {
 };
 ```
 
+## Using WebdriverIO
+
+[WebdriverIO](https://webdriver.io) is a good alternative for your component or end-to-end tests. It has very compelling advantages like support for [mocking](https://webdriver.io/docs/component-testing/mocking) and [code coverage](https://webdriver.io/docs/component-testing/coverage) reporting.
+
+You can set up WebdriverIO in your project via:
+
+```bash
+npm init wdio@latest ./
+```
+
+It will start a configuration wizard that will guide you through some questions. Make sure select the following:
+
+- __What type of testing would you like to do?__:<br />Component or Unit Testing - in the browser
```

Comment:
Did `prettier` put in the self closing tag? `<br />` -> `<br>`?

Reply:
Nope, I am using Docusaurus a lot which mdx parser requires closing tags, or at least used to. Shall I revert?

Reply:
`<br>` is a [void element](https://developer.mozilla.org/en-US/docs/Glossary/Void_element) so it can be represented as just `<br>` if you don't mind updating it.

Reply:
done